.travis.yml

```diff
@@ -2,7 +2,8 @@ language: ruby
 notifications:
   email: false
 rvm:
-  - 2.3.1
   - 2.4.0
```

Comment:
guessing we don't want to bother bumping the minor versions of 2.4 and 2.5?

Reply:
i see us removing support for this in the near future, so i figured it's not worth messing with